src/main/java/frc/robot/Subsystems/IntakePlacer.java

```diff
@@ -0,0 +1,69 @@
+package frc.robot.Subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.RobotMap;
+
+/**
+ * Controls the position of the {@code IntakeRoller}.
+ *
+ * @see MotoredGenericSubsystem
+ */
+public class IntakePlacer extends MotoredGenericSubsystem {
+
+    public static final double MAX_SPEED = 0.5;
```

Comment:
True for all subsystems:
If this is a hard mechanical limit on the motor, like giving 0.6 for 755 motors this is ok. If however this is just the speed for the commands this should be in the dashboard so you can change it at will if you see it's moving too slow or fast